src/main.js

```diff
@@ -114,8 +121,11 @@ const bot = new Client({
 
 bot.on("ready", async () => {
   // when loaded (ready event)
-  bot.user.setActivity(`${PREFIX}help | ${PREFIX}info`, { type: "PLAYING" });
+  bot.user.setActivity(config.activity.description, {
+    type: config.activity.type.toUpperCase(),
+  });
   utils.logger.log("debug", `${bot.user.username} is ready...`);
+  console.log(`${bot.user.username} is ready...`);
```

Comment:
Per the style guide, console.log should not be in production code. It's already being logged at the debug level. If you think it should be logged at a more severe level, update that instead.